pinot-core/src/main/java/org/apache/pinot/core/util/TableConfigUtils.java

```diff
@@ -65,6 +68,7 @@ public static void validate(TableConfig tableConfig, @Nullable Schema schema) {
     validateValidationConfig(tableConfig, schema);
     validateIngestionConfig(tableConfig.getIngestionConfig(), schema);
     validateTierConfigList(tableConfig.getTierConfigsList());
+    validateIndexingConfig(tableConfig.getIndexingConfig(), schema);
```

Comment:
Could you update the description of the method `validate` as well?

Reply:
Done